<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisualHandGestureRecognitionClass</class>
 <widget class="QMainWindow" name="VisualHandGestureRecognitionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1470</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VisualHandGestureRecognition</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_show_color">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_show_depth">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_show_hand_gesture">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>120</y>
      <width>151</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_hand_gesture_num">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>330</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1470</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="VisualHandGestureRecognition.qrc"/>
 </resources>
 <connections/>
</ui>
